src/Symfony/Component/Routing/Generator/UrlGenerator.php

```diff
@@ -76,6 +76,11 @@ class UrlGenerator implements UrlGeneratorInterface, ConfigurableRequirementsInt
     );
 
     /**
+     * @var string This regexp matches all characters that are not or should not be encoded by rawurlencode (see list in array above).
+     */
+    protected $urlEncodingSkipRegexp = '#[^-.~a-zA-Z0-9_/@:;,=+!*|]#';
```

Comment:
@stof It could be, but all other properties are protected as well, so I'd defiate from the 'default' in that class.